ingesters/collectd.md

```diff
@@ -162,3 +193,99 @@ Tag-Plugin-Override=memory:memstats # Map the memory plugin data to the "memstat
 Tag-Plugin-Override= df : diskdata  # Map the df plugin data to the "diskdata" tag.
 Tag-Plugin-Override = disk : diskdata  # Map the disk plugin data to the "diskdata" tag.
 ```
+
+
+## Example Collect Configuration
+
+The Collectd system is a plugin based system instrumentation and testing framework, there is no standard Collectd deployment and every plugin can send a unique set of fields and structures.   The only hard requirement for configuring the Collectd system with Gravwell is a proper `network` Plugin definition with matching username, password, and Security-Level.  Here are two basic configurations that will collect some reasonable metrics and send them to Gravwell:
```

Comment:
When I first read "two basic configurations," I was expecting 2 examples. It may be worthwhile to edit this to say something like "Here is an example of a plugin configuration and a collectd configuration that work together to collect some reasonable metrics and send them to Gravwell:"